README.md

```diff
@@ -145,14 +145,18 @@ the shore, interacting with the different elements of the bathymetry.
 
 #### Example
 
-Initialize the scenario:
+Initialize a bathymetry from a given text file and the coastal area scenario:
 
 ```python
 from inductiva import fluids
 
+bathymetry = fluids.scenarios.Bathymetry(text_file_path="bathymetry.txt", 
+                                         x_range=(0, 1200), y_range=(0, 400))
```

Comment:
Please see the `Bathymetry` class. You need to use the classmethod `from_text_file` to load a bathymetry from a file...

Reply:
I just copied from the PR... I will add that.

Reply:
Sorry I missed it in the PR description.